.github/ISSUE_TEMPLATE/issue_template.md

```
@@ -0,0 +1,23 @@
+---
+name: Android ISSUE_TEMPLATE
+about: 버그, 기능 제안 등 작업 항목을 등록해주세요.
+title: "[TYPE] 작업 내용"
+labels: ''
+assignees: ''
+
+---
+
+## 작업 유형
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)